File: src/question/day0703/ms_0703.cs
using System;

namespace algorithm_05
{
    class ms_0703
    {
        static void Main(string[] args)
        {
            string[] area = Console.ReadLine().Split(new char[] {' '});
            var width = Convert.ToInt32(area[0]) + 5;
            var length = Convert.ToInt32(area[1]) * 2;

            Console.WriteLine($"width = {width}\nlength = {length}");
            Console.WriteLine($"area = {width * length}");
        }
    }
}

File: year2020/month07/day0703/ms_0703.cpp
#include <iostream>

using namespace std;

int main() {
	int width;
	int length;

	cin >> width >> length;

	width += 5;
	length *= 2;

	cout << "width = " << width << endl;
	cout << "length = " << length << endl;
	cout << "area = " << width * length << endl;
 
	return 0;
}
